backend/tests/integration-async/records/indexer/test_literature.py

```diff
@@ -239,12 +239,14 @@ def assert_record():
     citing_record.update(data_citing_record)
     db.session.commit()
 
+    expected_citation_year = [{"count": 1, "year": 2019}]
+
     def assert_record():
         current_search.flush_and_refresh("records-hep")
         record_from_es = LiteratureSearch().get_record_data_from_es(cited_record)
         assert expected_citation_year == record_from_es["citations_by_year"]
```

Comment:
`expected_citation_year` was None?

Reply:
No, it was the one from before the update ;)